src/main/webapp/i18n/de/exam.json

```diff
@@ -4,7 +4,22 @@
             "title": "Exam"
         },
         "examManagement": {
-            "title": "Klausur Verwaltung"
+            "title": "Klausur Verwaltung",
```

Comment:
Is "Klausur" a good term here? I think "Prüfung" would be better since "Klausur" is imo a reserved term.

Reply:
Not really important right now imo. Working prototype is more important we can discuss such translation issues later